### PR TITLE
Ship.js Integration for vue 3 branch

### LINKED ISSUE #949: Shipjs Integration

### PR DESCRIPTION
**Pull request by subzero10**
Fixes #949.

### CODE REVIEW

File: .github/workflows/shipjs-manual-prepare.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    if: |
+      github.event_name == 'issue_comment' &&
+      (github.event.comment.author_association == 'member' || github.event.comment.author_association == 'owner') &&
+      startsWith(github.event.comment.body, '@shipjs prepare')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: vue3
```
Comment (subzero10) [inline .github/workflows/shipjs-manual-prepare.yml:16]: `vue3` instead of `master`.

File: .github/workflows/shipjs-schedule-prepare.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    # * is a special character in YAML so you have to quote this string
+    - cron:  "0 8 * * 2"
+jobs:
+  schedule_prepare:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: vue3
```
Comment (subzero10) [inline .github/workflows/shipjs-schedule-prepare.yml:13]: `vue3` instead of `master`.

File: .github/workflows/shipjs-trigger.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true && startsWith(github.head_ref, 'releases/v')
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: vue3
```
Comment (subzero10) [inline .github/workflows/shipjs-trigger.yml:15]: `vue3` instead of `master`.

File: ship.config.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  shouldPrepare: ({ releaseType, commitNumbersPerType }) => {
+    const { fix = 0 } = commitNumbersPerType;
+    if (releaseType === 'patch' && fix === 0) {
+      return false;
+    }
+    return true;
+  },
+  publishCommand: ({ defaultCommand, tag }) =>
+    `${defaultCommand} --access public --tag next`,
```
Comment (subzero10) [inline ship.config.js:10]: I couldn't find a way to let ship.js know that the tag should always be `next`, so I am just setting it here manually.

**Review by joshuap — APPROVED** [review]

**Comment by subzero10** [comment]
Done. Since CI only runs on main branch, I deployed manually:
After merging this PR to vue3 branch:
- `git checkout vue3`
- `GITHUB_TOKEN=ABC npm run release`
 
After merging release PR (created from previous command) to vue3 branch:
- `git checkout vue3`
- `NPM_AUTH_TOKEN=ABC shipjs trigger`

**Pull request merged**